frontend/src/scenes/settings/environment/SessionRecordingSettings.tsx

```diff
@@ -484,6 +607,7 @@ export function ReplayCostControl(): JSX.Element | null {
                     </>
                 )}
                 <LinkedFlagSelector />
+                <UrlTriggerOptions />
```

Comment:
we could wrap this in a `FlaggedFeature` so we can roll it out just for team 2 at first so we can get a little testing done in prod